distribution/bin/testgrid

```diff
@@ -0,0 +1,185 @@
+  # Set job priority to standard for interactive (interactive - 6) by using
+  # the interactive priority - 6, the helper threads that respond to requests
+  # will be running at the same priority as interactive jobs.
+  COMMAND='chgjob job('$JOBNAME') runpty(6)'
+  system $COMMAND
+
+  # Enable multi threading
+  QIBM_MULTI_THREADED=Y
+  export QIBM_MULTI_THREADED
+fi
+
+# For Migwn, ensure paths are in UNIX format before anything is touched
+if $mingw ; then
+  [ -n "$TESTGRID_DISTRIBUTION" ] &&
+    TESTGRID_DISTRIBUTION="`(cd "$TESTGRID_DISTRIBUTION"; pwd)`"
+  [ -n "$JAVA_HOME" ] &&
+    JAVA_HOME="`(cd "$JAVA_HOME"; pwd)`"
+fi
+
+if [ -z "$JAVACMD" ] ; then
+  if [ -n "$JAVA_HOME"  ] ; then
+    if [ -x "$JAVA_HOME/jre/sh/java" ] ; then
+      # IBM's JDK on AIX uses strange locations for the executables
+      JAVACMD="$JAVA_HOME/jre/sh/java"
+    else
+      JAVACMD="$JAVA_HOME/bin/java"
+    fi
+  else
+    JAVACMD=java
+  fi
+fi
+
+if [ ! -x "$JAVACMD" ] ; then
+  echo "Error: JAVA_HOME is not defined correctly."
+  exit 1
+fi
+
+# if JAVA_HOME is not set we're not happy
+if [ -z "$JAVA_HOME" ]; then
+  echo "You must set the JAVA_HOME variable before running Testgrid."
+  exit 1
+fi
+
+# if TESTGRID_HOME is not set we're not happy
+if [ -z "$TESTGRID_HOME" ]; then
+  echo "TESTGRID_HOME is not set, system will setup a default location"
+else
+  echo "Testgrid Home is set to : " $TESTGRID_HOME
+fi
+
+# ----- Process the input command ----------------------------------------------
+
+for c in "$@"
+do
+    if [ "$c" = "--debug" ] || [ "$c" = "-debug" ] || [ "$c" = "debug" ]; then
+          CMD="--debug"
+    elif [ "$CMD" = "--debug" ] && [ -z "$PORT" ]; then
+          PORT=$c
+    fi
+done
+
+if [ "$CMD" = "--debug" ]; then
+  if [ "$PORT" = "" ]; then
+    echo "Please specify the debug port after the --debug option"
+    exit 1
+  fi
+  if [ -n "$JAVA_OPTS" ]; then
+    echo "Warning !!!. User specified JAVA_OPTS will be ignored, once you give the --debug option."
+  fi
+  JAVA_OPTS="-Xdebug -Xnoagent -Djava.compiler=NONE -Xrunjdwp:transport=dt_socket,server=y,suspend=y,address=$PORT"
+  echo "Please start the remote debugging client to continue..."
+fi
+
+# For Cygwin, switch paths to Windows format before running java
+if $cygwin; then
+  JAVA_HOME=`cygpath --absolute --windows "$JAVA_HOME"`
+  TESTGRID_DISTRIBUTION=`cygpath --absolute --windows "$TESTGRID_DISTRIBUTION"`
+  CLASSPATH=`cygpath --path --windows "$CLASSPATH"`
+  JAVA_ENDORSED_DIRS=`cygpath --path --windows "$JAVA_ENDORSED_DIRS"`
+fi
+
+# Setting the classpath
+export CLASSPATH=$TESTGRID_DISTRIBUTION/lib/*
+
+# ----- Execute The Requested Command -----------------------------------------
+
+$JAVACMD \
+	-Xms256m -Xmx1024m \
+	-XX:+HeapDumpOnOutOfMemoryError \
+	-XX:HeapDumpPath="$TESTGRID_DISTRIBUTION/heap-dump.hprof" \
+	$JAVA_OPTS \
+	-Djava.security.egd=file:/dev/./urandom \
+	-Dfile.encoding=UTF8 \
+	-Dlog4j.configurationFile=$TG_DIST_HOME/log4j2.xml \
```

Comment:
I think `TG_DIST_HOME` is renamed to `TESTGRID_DISTRIBUTION` now.

Reply:
Yes, This is renamed.